_ide_helper.php

```
@@ -5191,7 +5233,32 @@
         /**
-         * Specify the name of the connection that should be used to manage locks.
+         * Set the underlying database connection.
+         *
+         * @param \Illuminate\Database\ConnectionInterface $connection
+         * @return \Illuminate\Cache\DatabaseStore
+         * @static
+         */
+        public static function setConnection($connection)
+        {
+            /** @var \Illuminate\Cache\DatabaseStore $instance */
+            return $instance->setConnection($connection);
```

PHPMD (2.15.0)

Avoid using undefined variables such as '$instance' which will lead to PHP notices. (Clean Code Rules)

(UndefinedVariable)